test/config/utility.cc

```diff
@@ -12,7 +12,35 @@
 
 namespace Envoy {
 
-const std::string basic_config = R"EOF(
+const std::string ConfigHelper::TCP_PROXY_CONFIG = R"EOF(
+static_resources:
+  listeners:
+    name: listener_0
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 0
+    filter_chains:
+      filters:
+        name: envoy.tcp_proxy
+        config:
+          stat_prefix: tcp_stats
+          cluster: cluster_0
+  clusters:
+    name: cluster_0
+    hosts:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 0
+admin:
```

Comment:
Out of curiosity, could we have some common base config (admin, clusters) for HTTP and TCP testing, which gets the HTTP and TCP filter chains added later? I don't think it's necessarily worth doing here, but if it makes sense, a TODO to refactor could work.

Reply:
Hm, reduces duplication but also reduces the mono-simple-config Matt asked form.
@mattklein123 any preference on this one?

Reply:
If it's all inside this file and you prefer to split it up that's fine with me. I do find it generally easier to look at config blocks and I'm not sure how much DRYing in test code is really worth it if it makes readability worse. But like I said either way is fine with me.